app/src/main/java/com/zendesk/rememberthedate/push/RegistrationIntentService.java

```diff
@@ -69,7 +69,7 @@ void enablePush(final PushNotificationStorage pushNotificationStorage){
             @Override
             public void onSuccess(final String result) {
 
-                final AuthenticationType authentication = ZendeskConfig.INSTANCE.getSettings().getSdkSettings().getAuthentication();
+                final AuthenticationType authentication = ZendeskConfig.INSTANCE.getMobileSettings().getAuthenticationType();
```

Comment:
Never mind, it's listed here: https://github.com/zendesk/zendesk_sdk_android/blob/6d3c62f1014f5769eb03da65252c8a1d26adbcb2/docs/migration.md#L77-L77